CodeEdit/Utils/ShellClient/Models/ShellClient.swift

```diff
@@ -65,8 +70,10 @@ class ShellClient {
                     subject.send(completion: .finished)
                     return
                 }
-                String(decoding: data, as: UTF8.self)
-                    .split(whereSeparator: \.isNewline)
+                guard let output = String(bytes: data, encoding: .utf8) else {
+                    return
```

Comment:
Needs to tell the `subject` it's done since we aren't throwing here.

```suggestion
                    subject.send(completion: .finished)
                    return
```

Reply:
Ahh, right. Done!